properties.schema

```diff
@@ -3,6 +3,26 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "id": "http://jsonschema.net",
   "required": false,
+  "globals": {
+    "resumeButtonLabel": {
+      "type": "string",
+      "required": true,
+      "default": "Resume",
+      "inputType": "Text",
+      "validators": [],
+      "translatable": true,
+      "help": "Label for ubiquitous resume button."
+    },
+    "resumeButtonAriaLabel": {
+      "type": "string",
+      "required": true,
+      "default": "Navigate to your furthest point of progress.",
+      "inputType": "Text",
+      "validators": [],
+      "translatable": true,
+      "help": "ARIA label for ubiquitous resume button."
```

Comment:
Help text isn't necessary here. Use of the word ubiquitous is perhaps a little much for non-English speakers.
```suggestion
```